packages/apps/reputation-oracle/server/src/modules/auth/auth.controller.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  Body,
+  ClassSerializerInterceptor,
+  Controller,
+  HttpCode,
+  Post,
+  Req,
+  UseGuards,
+  UseInterceptors,
+} from '@nestjs/common';
+
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { Public } from '../../common/decorators';
+import { UserCreateDto } from '../user/user.dto';
+import {
+  AuthDto,
+  ForgotPasswordDto,
+  ResendEmailVerificationDto,
+  RestorePasswordDto,
+  SignInDto,
+  VerifyEmailDto,
+} from './auth.dto';
+import { AuthService } from './auth.service';
+import { JwtAuthGuard } from '../../common/guards';
+import { RequestWithUser } from '../../common/types';
+
+@ApiTags('Auth')
+@Controller('/auth')
+export class AuthJwtController {
+  constructor(private readonly authService: AuthService) {}
+
+  @Public()
+  @Post('/signup')
+  @UseInterceptors(ClassSerializerInterceptor)
+  public async signup(@Body() data: UserCreateDto): Promise<void> {
```

Comment:
UserCreateDto cannot include the status, in the sign up, the user could set any status here.

Reply:
How should we do it? Assigning Operator by default and then the user type needs to be updated?